README.md

```diff
@@ -1,18 +1,18 @@
+1 | Polarization | <ul><li> Project group accommodates some rather than all representatives; favours appearances over outcomes.</ul></li><ul><li>Volume of work is prioritized over efficiency and organization.</ul></li><ul><li>Feedback process follows antiquated methodology, is irregular, non-existant or biased, only being initiated if a large problem is brought forward or a large investment is involved; feedback and areas of improvement from minorities are either minimalized, avoided, and/or ignored.</ul></li><ul><li>"Walking on Thin Ice" approach is common as potentially sensitive project elements, such as accessibility and demographic data, is regarded with fear of open conflict - inequalities and large data gaps are pravalent.</ul></li><ul><li>Product/Service data are collected from random sample(s) of users instead of all users who maintain, use and implement the product/service.</ul></li><ul><li>Records management is acknowledged and approached with simplicity rather than complexity such as consideration of electronic EDRMS folders, off-site records maintenance, different types of storage, etc.</ul></li>
+2 | Minimization | <ul><li>Use of Produce/Service highlights oppressive policies and law, privacy and security, as well as network infrastructure</ul></li> <ul><li>Semantics and syntax of primary language accommodates mainly majority group, and at most, minimal minority groups</ul></li><ul><li> Consultation for anti-oppressive language is in progress and data analysts are engaged for optimizing clean and race based data</ul></li><ul><li> Process for feedback begins and complex decision making is in development.</ul></li> <ul><li>The working group has minimal representatives of miniority groups</ul></li>
```

Comment:
Periods have dropped off from the end of bullets for sections 2-4.

Reply:
Sections 2 onward are WIP so I will maintain until next draft releases.